### PR TITLE
Add Developer Sprints 2018 blog

### LINKED ISSUE #145: Re-write the blog for PyConIndia 2018 Devsprints
Use this for reference - https://in.pycon.org/blog/2017/understanding-dev-sprints.html

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Fixes #145 
@inovizz  Need some more information, mentioned it with capital letters.
@pythonindia/pycon-2018-team Review needed

### CODE REVIEW

**Comment by inovizz** [comment]
I guess you'd have to park this PR for some time and let CFP team come up with a plan on how to take proposals for DevSprints and what is the deadline.

I've already notified them about this.

**Comment by vipulgupta2048** [comment]
I don't mind closing it if it is necessary, though I appreciate if the changes are not big then they could edit this PR since some work has already been done, I have opened the edit from maintainers.

**Comment by vipulgupta2048** [comment]
@ananyo2012  @inovizz  Please review.

File: content/blog/understanding-dev-sprints-2018.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<!-- PELICAN_END_SUMMARY -->
+
+PyCon India 2018 is a place for developers from across across the country (and beyond) to get together and celebrate Python. But what good is a PyCon without getting your hands dirty writing some code? Hence, for all Open Source enthusiasts out there, the PyCon India team is happy to announce Dev Sprints @ PyCon India 2018.
+
+During Dev sprints, Mentors/Contributors of OSS projects guide participants through the end to end process of OSS contribution, right from setting up build environments, to developing and testing code, to submitting patches for bugs. If you are stuck, mentors are all available to help you figure out issues and get you back up and running! From past experiences, we’ve seen a lot of people start off contributing to OSS through Dev Sprints and later go on to become regular contributors to various projects across the web.
+
+> Dev sprints are for everybody who want to participate and learn, as issues on repositories don't discriminate.
```
Comment (realslimshanky) [inline content/blog/understanding-dev-sprints-2018.md]: :tada: This rocks!

Comment (vipulgupta2048) [reply content/blog/understanding-dev-sprints-2018.md]: Thank you :heart:

**Pull request merged**